index.bs

```diff
@@ -1402,6 +1405,87 @@ lt="ReadableStreamDefaultReader(stream)">new ReadableStreamDefaultReader(<var>st
   1. Perform ! ReadableStreamReaderGenericRelease(*this*).
 </emu-alg>
 
+<h5 id="default-reader-iterator" method for="ReadableStreamDefaultReader">
+  iterator({ cancel = true } = {})
+</h5>
+
+<div class="note">
+  The <code>iterator</code> method returns an async iterator which can be used to consume the stream. The return method
+  of this iterator will optionally release or cancel this reader.
```

Comment:
1. this iterator object
2. Link [release](https://streams.spec.whatwg.org/#release-a-lock) and maybe [cancel](https://streams.spec.whatwg.org/#cancel-a-readable-stream) too?